### PR TITLE
Pylint fixes for webapp extension

### LINKED ISSUE #166: Fix webapp pylint errors
### Webapp 

Run ./scripts/ci/test_static.sh on the [pylint](https://github.com/Azure/azure-cli-extensions/tree/pylint) branch
Pylint violations: https://travis-ci.org/Azure/azure-cli-extensions/jobs/375143935

-----

### PR DESCRIPTION
**Pull request by panchagnula**
Fixes  #166
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md).

### CODE REVIEW

File: src/webapp/azext_webapp/tunnel.py [hunk 1]
```diff
@@ -159,7 +164,8 @@ def listen_to_web_socket(self, client, ws_socket, index):
                 ws_socket.close()
                 return False
 
-    def listen_to_client(self, client, ws_socket, index):
+# pylint: disable=no-self-use
```
Comment (LukaszStem) [inline src/webapp/azext_webapp/tunnel.py]: Please align this comment

File: src/webapp/azext_webapp/tunnel.py [hunk 2]
```diff
@@ -136,7 +140,8 @@ def listen(self):
             logger.info('Both debugger and websocket threads stopped...')
             logger.warning('Stopped local server..')
 
-    def listen_to_web_socket(self, client, ws_socket, index):
+# pylint: disable=no-self-use
```
Comment (LukaszStem) [inline src/webapp/azext_webapp/tunnel.py]: Please align this with the corresponding method

File: src/webapp/azext_webapp/tunnel.py [hunk 3]
```diff
@@ -34,6 +34,7 @@ def recv(self):
         return data
 
 
+# pylint: disable=no-member, too-many-instance-attributes,bare-except,
```
Comment (LukaszStem) [inline src/webapp/azext_webapp/tunnel.py]: Please space correctly and remove last ','

File: src/webapp/azext_webapp/tunnel.py [hunk 4]
```diff
@@ -62,8 +65,9 @@ def create_basic_auth(self):
 
     def is_port_open(self):
         is_port_open = False
+        # pylint: disable=no-member
```
Comment (williexu) [inline src/webapp/azext_webapp/tunnel.py]: No need for another disable no-member here if it is already disabled for the class (line 37)

**Comment by williexu** [comment]
@panchagnula Thanks for the quick fixes :)

**Pull request merged**